cmd/helm/root.go

```diff
@@ -44,10 +44,13 @@ Common actions from this point include:
 - helm list:      list releases of charts
 
 Environment:
-  $HELM_HOME          set an alternative location for Helm files. By default, these are stored in ~/.helm
+  $HELM_HOME          set an alternative location for Helm files. By default, these are stored in
+                      "$XDG_CONFIG_DIR/helm" ("$HOME/.config/helm" on Linux,
+                      "%APPDATA%\helm" on Windows and "$HOME/Library/Preferences" on OSX).
+                      NOTE: if there is old-style "$HOME/.helm" directory, it will be used.
```

Comment:
Can you please remove the quotes around the environment variables?

```
$HELM_HOME          set an alternative location for Helm files. By default, these are stored in
                      $XDG_CONFIG_DIR/helm ($HOME/.config/helm on Linux,
                      %APPDATA%\helm on Windows and $HOME/Library/Preferences on OSX).
                      NOTE: if $HOME/.helm exists, it will be used instead of $XDG_CONFIG_HOME/helm.
```